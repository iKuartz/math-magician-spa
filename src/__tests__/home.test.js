import renderer from 'react-test-renderer';
import Home from '../components/Home';

it('Checking if Home page renders correctly', () => {
  const tree = renderer
    .create(
       <Home />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
