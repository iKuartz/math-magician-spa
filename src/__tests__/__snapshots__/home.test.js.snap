// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checking if Home page renders correctly 1`] = `
<div
  className="mainDiv"
>
  <h2
    className="welcome"
  >
    Welcome to our page!
  </h2>
  <p
    className="textParagraph"
  >
    In the words of Sofia Kovalevskaya: many who have had an opportunity of knowing any more about mathematics confuse it with arithmetic, and consider it an arid science. In reality, however, it is a science which requires a great amount of imagination.
  </p>
  <p
    className="textParagraph"
  >
    As said by Marston Morse: Mathematics are the result of mysterious powers which no one understands, and which the unconscious recognition of beauty must play an important part. Out of an infinity of designs a mathematician chooses one pattern for beauty's sake and pulls it down to earth.
  </p>
</div>
`;
